internal/ledgerutil/compare.go

```diff
@@ -23,48 +23,67 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	resultsDir      = "ledger_comparison_results"
+	allDivsFilename = "all_divergences.json"
```

Comment:
I understand that we would produce more than one file, e.g.,  `all_divergences.json` would contain all diversions and perhaps `top_earliest_divergences.json` would contain top-10. But I am not sure, what is the significant of one extra dir `ledger_comparison_results`. I think that you can produce the output files directly in the user supplied outDir

Reply:
I feel that creating a directory for both results organizes everything a lot better. This is similar to how the snapshot tool works. Maybe we can get some more opinions on this from other maintainers?

Reply:
Not sure which snapshot tool are you referring to. Assuming you are referring to the snapshot generation, all the files are produced in a single folder.